server/src/models/blogs.js

```diff
@@ -6,31 +6,44 @@ const BlogsSchema = new Schema({
   userId: {
     type: Schema.Types.ObjectId,
     ref: 'User',
+    index: true,
   },
   title: {
     type: String,
     required: true,
+    default: 'My Blog',
   },
-  content: [
-    {
-      _id: Schema.Types.ObjectId,
-      type: String,
-      fragment: {
-        type: String,
-        required: true,
-      },
-    },
-  ],
-  likes: {
-    type: [Schema.Types.ObjectId],
-    ref: 'User',
+  content: {
+    type: String,
+    required: true,
+    default: '## My Blog is about',
```

Comment:
if it is required what is the point of default?

Reply:
Ohh! Yes required is not really important here. If you want I can remove this. But this will do no harm according to me :smile: